docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+services:
+  web:
+    build: .
+    ports:
+      - "8080:8080"
+    environment:
+      - JAVA_OPTS=-Dspring.data.mongodb.uri=mongodb://db:27017/session-service
+      # with user/pass enabled:
+      # - JAVA_OPTS=-Dspring.data.mongodb.uri=mongodb://db:27017/session-service -Dsecurity.basic.enabled=true -Dsecurity.user.name=user -Dsecurity.user.password=pass
+    links:
+      - db
+    depends_on:
+      - db
+  db:
+    image: mongo:3.6.6
+    restart: always
```

Comment:
can we also have the following:

- container a name e.g cbio_session_mongodb
- export port 27017
- run time options such as using --smallfiles

Reply:
@khzhu thanks for reviewing!

- it automatically gets a name assigned based on the folder it is in e.g. for me `session-service_db_1`. That seems specific enough right?
- im not sure if it's a good idea to expose 27017 by default. If you run that on your server and the port is open to the world it could be a security risk. One can connect to the container directly instead: `docker exec -it session-service_db_1 mongo mongodb://localhost:27017`. Will add to docs
- Will add that runtime option

Reply:
@inodb ,I mean map port 27017 of local machine to container 27017 to make mongodb accessible for some IDE like NoSQL Booster. Not open 27017 to the internet.

Reply:
container name you have is okay.

Reply:
@khzhu i've added it to the docker-compose file with some comments in the README as well. I think it's better to default to not exposing it to localhost, but allow people to explicitly do so if they want to. It's not necessary for the app to work. Being able to connect locally is more of a debugging/dev feature

Reply:
okay, convinced. thanks!